server/container/guice/cassandra-guice/src/test/java/org/apache/james/DockerElasticSearchExtension.java

```diff
@@ -19,31 +19,23 @@
 
 package org.apache.james;
 
+import org.apache.james.backends.es.DockerElasticSearch;
+import org.apache.james.backends.es.DockerElasticSearchSingleton;
 import org.apache.james.backends.es.ElasticSearchConfiguration;
-import org.apache.james.util.Host;
-import org.apache.james.util.docker.DockerGenericContainer;
 import org.junit.jupiter.api.extension.ExtensionContext;
 
 import com.google.inject.Module;
 
 public class DockerElasticSearchExtension implements GuiceModuleTestExtension {
-    public static final int ELASTIC_SEARCH_PORT = 9300;
-    public static final int ELASTIC_SEARCH_HTTP_PORT = 9200;
-
-    private final DockerGenericContainer elasticSearchContainer;
-
-    public DockerElasticSearchExtension(DockerGenericContainer elasticSearchContainer) {
-        this.elasticSearchContainer = elasticSearchContainer;
-    }
 
     @Override
     public void beforeEach(ExtensionContext extensionContext) {
-        elasticSearchContainer.start();
+        DockerElasticSearchSingleton.start();
```

Comment:
Can you try with `DockerElasticSearchSingleton.INSTANCE.start()` so that you don't need the empty start method there? This is what is done in Cassandra....